guides/common/modules/proc_cloning_satellite_server.adoc

```diff
@@ -168,8 +168,9 @@ You can either mount the shared storage or copy the backup files to the `/backup
 # subscription-manager attach --pool=__pool_ID__
 # subscription-manager repos --disable=*
 # subscription-manager repos --enable={RepoRHEL8AppStream} \
---enable={RepoRHEL8ServerSatelliteServerProductVersion} \
+--enable={RepoRHEL8BaseOS} \
 --enable={RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+# dnf module enable satellite-maintenance:el8
```

Comment:
Ideally, we'd have each command as a separate step but that is definitely a work for another PR.

Reply:
@adamlazik1 I'll add the step that Eric mentioned in this PR itself. I agree with you that these should be separate steps, I'll push a PR for this soon :)